docker/install_node.sh

```diff
@@ -46,14 +54,20 @@ else
     mkdir -p ../oppia_tools
     tar -xvf node-download -C ../oppia_tools
     rm node-download
-fi
 
-if [ "$node_file_name" = "node-v16.13.0" ]; then
-    cd ../oppia_tools/node-16.13.0
-    ./configure
-    make
+    # Build node.js if it is installed using source code (more info https://github.com/nodejs/node/blob/main/BUILDING.md#building-nodejs-1).
```

Comment:
Update the link to https://github.com/nodejs/node/blob/v16.x/BUILDING.md Since we;re using node v16

Reply:
done!